src/common/constants.js

```diff
@@ -48,6 +48,7 @@ export const dp2p_google_play_url =
 export const gtm_test_domain = 'deriv-com.binary.sx'
 export const livechat_client_id = '66aa088aad5a414484c1fd1fa8a5ace7'
 export const livechat_license_id = 12049137
+export const deriv_app_app_id = domain_url === deriv_me_url ? 1411 : 16929
```

Comment:
I have small change request. Let's save the app_id's in descriptive variable, and use in this line.  And we can use 'deriv_com_url' as primary check.

Reply:
I've applied the suggestions~ @ShuvoHabib do check if it is clear enough, thanks 😄

Reply:
Perfect, mate 😄